<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGameDialog</class>
 <widget class="QDialog" name="QtGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>City of Thieves</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: black;
  border: 1px solid lime
}

QPlainTextEdit {
  font: 32px  monospace;
  color: lime;
  background-color: black;
  border: 1px solid green
}

QLabel {
  font: 18px  monospace;
  border: 1px solid black;
  color: lime;
  background-color: black
}

QListWidget{
  font: 18px  monospace;
  border: 1px solid lime;
  color: lime;
  background-color: black
}


QScrollBar {
  color: lime;
  background-color: black;
  border: 1px solid lime
}

QScrollBar::handle {
  background: green
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>100</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
     <property name="centerOnScroll">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_luck">
        <property name="text">
         <string>Luck:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_gold">
        <property name="text">
         <string>Gold: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_condition">
        <property name="text">
         <string>Condition:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="image">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="QtResources.qrc">:/images/ZanbarBone.jpg</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_skill">
        <property name="text">
         <string>Skill:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_provisions">
        <property name="text">
         <string>Provisions:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QListWidget" name="list_items">
        <item>
         <property name="text">
          <string>(no items yet)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="QtResources.qrc"/>
 </resources>
 <connections/>
</ui>
